src/Microsoft.DocAsCode.Build.Common/ModelAttributeHandlers/Handlers/MarkdownContentHandler.cs

```diff
@@ -66,50 +90,80 @@ protected override void HandleCurrentProperty(object declaringObject, PropertyIn
                 }
             }
 
-            protected override PropInfo[] GetProps(Type type)
+            protected override void HandleDictionaryType(object declaringObject, PropertyInfo currentPropertyInfo, HandleModelAttributesContext context)
             {
-                return (from prop in ReflectionHelper.GetSettableProperties(type)
-                        let attr = prop.GetCustomAttribute<MarkdownContentAttribute>()
-                        select new PropInfo
+                var type = currentPropertyInfo.PropertyType;
+                if (type.GetGenericTypeDefinition() == typeof(Dictionary<,>))
```

Comment:
Dictionary<string, object>.GetGenericTypeDefinition() is Dictionary<,>

---
In reply to: [95296807](https://github.com/dotnet/docfx/pull/1137#discussion_r95296807) [](ancestors = 95296807)